outreach/sops/README.md

```diff
@@ -0,0 +1,41 @@
+# Operation Code Outreach Team
+
+Thanks for joining the Operation Code Outreach Team!
+
+## Tools and Communication
+
+At this time, the Outreach team primarily utilizes Slack for communication - specifically the `#greetings` and `#outreach` channels. If you are not in these channels, please reach out to someone on the team or in Operation Code leadership to ensure you are added. The `#outreach` channel is where most information is shared, and it's the best place to voice any issues and concerns or ask any questions that might come up!
+
+There is also a monthly sync-up meeting Outreach team members are encouraged to attend. Meeting invites are sent via Google Calendar invite. Please let someone on the team know if you have not received an invitation.
+
+## Workflow
+
+This is the typical workflow that the Outreach team follows when reaching out to our members. It is NOT a concrete process, and feedback is welcome! (and encouraged!)
+
+### New Member Outreach
+
+1. When the member joins, a notification will show up in `#greetings`
+2. Look for the notification that says ` Outreach Team: Please reach out to NAME via DM` 
+3. Direct message the new member - personalize the welcome! Create a great experience and ensure they have the resources and/or connections they need to be successful!
+4. After you DM the new member, click the `I will DM them!` button.
+5. Head over to `#outreach` and set a reminder to contact them after about a month into their membership.
+    - In the `#outreach` channel, type something in the chatbox similar to `/remind #outreach to contact NAME in 30 days`. You can put your own spin on this, or you can copy and paste that line
+
+### Member 30-day Follow-up
+
+1. A notification will pop up in `#outreach` - the specific message will vary depending on what team member set the reminder, but it will advise to contact the member.
+2. Reach out to the member! Check in, have a conversation, see how things are going. Share some tips, or refer them to a resource or another OC member, mentor, or staff if needed.
```

Comment:
Thinking about ways we can help support this for the outreach team - giving you more resources to point folks to, maybe linking this to some of the FAQ work that's being done

Reply:
Yes absolutely. The faq and other resources would be great information to share. There could be opportunity to share more of what's available through OC so the team has the tools to set up new members for success.